R/grid_helpers.R

```diff
@@ -351,6 +370,17 @@ compute_grid_info <- function(workflow, grid) {
   # Now make a similar iterator across models. Conditioning on each unique
   # preprocessing candidate set, make an iterator for the model candidate sets
   # (if any)
+  if (any_parameters_postprocessor) {
+    # Ensure that the submodel trick kicks in by temporarily nesting the
+    # postprocessor parameters while iterating in the model grid
+    # TODO: will this introduce issues when there are matching postprocessor
+    # values across models?
+    # ... i think we actually want to (temporarily?) situate these as submodels
```

Comment:
My first pass at enabling the submodel trick—nest by the postprocessor and unnest later—but that doesn't quite do the job.

Reply:
> that doesn't quite do the job.

Can you expand on this?

Reply:
See the linked notes above!